app/views/home/index.html.erb

```diff
@@ -0,0 +1,11 @@
+
+<link href="https://fonts.googleapis.com/css?family=Poor+Story" rel="stylesheet">
```

Comment:
This is more appropriate in the `application.html.erb` file than in a specific view file